graphql/database/base/RedisModel.js

```diff
@@ -179,9 +164,8 @@ class RedisModel extends Model {
       const validateResult = this.schema[field].validate(fieldValue, {
         convert: true,
       })
-      if (validateResult.error) {
-        throw validateResult.error
-      }
+      // todo: @jedtan Implement Solution here for RedisModels
```

Comment:
This is because the `convert: true` parameter doesn't convert numbers -> strings properly. I don't think this is going to cause any immediate issues except for IDs occasionally being numbers in memory instead of strings (if a nanoid/uuid happens to be numeric by chance).